neurodamus/io/sonata_config.py

```diff
@@ -361,7 +359,7 @@ def parsedConnects(self):
         }
 
     @property
-    def parsedStimuli(self):
+    def parsedStimuli(self) -> list:
```

Comment:
docstring is missing and the function is not completely trivial

Reply:
Done